com.microsoft.mrtk.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -140,6 +140,11 @@ public override bool isHoverActive
             }
         }
 
+        [SerializeReference]
+        [InterfaceSelector]
+        [Tooltip("The pose source representing the ray this interactor uses for rotation")]
+        protected IPoseSource devicePoseSource;
```

Comment:
Looks like this one isn't set in the prefab

![image](https://user-images.githubusercontent.com/3580640/186006233-8203bc6a-d5da-484c-ab44-d48214ada8e7.png)

Reply:
Fixed along with reasonable inspector views (though I had to wrap it in a struct, attempts at making a custom editor were quite cumbersome)